frontend/src/metabase/components/ExplicitSize.jsx

```diff
@@ -88,15 +89,15 @@ export default ({ selector, wrapped } = {}) => ComposedComponent =>
       }
     }
 
-    _updateSize = () => {
+    _updateSize = _.throttle(() => {
```

Comment:
Nitpick: probably worthwhile injecting a one-liner comment here justifying the throttle.